cluster-operator/src/main/resources/cruiseControlDefaultLoggingProperties

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
```

Comment:
If this is a properties file, why doesn't it have the standard `.properties` file extension?

Reply:
Following the naming format of the other logging properties files (e.g. kafkaBridgeDefaultLoggingProperties, kafkaConnectDefaultLoggingProperties, entityTopicOperatorDefaultLoggingProperties,..). Is this incorrect? If so, I can add them to the files